packages/api-clients/graphql-client/src/graphql-client/types.ts

```diff
@@ -86,10 +89,8 @@ export interface ClientConfig {
   readonly retries: Required<ClientOptions>['retries'];
 }
 
-export interface RequestOptions {
+export interface RequestOptions extends Omit<CustomRequestInit, 'body'> {
   variables?: OperationVariables;
-  url?: string;
```

Comment:
Do you mean to change the `RequestOptions` interface or just add `signal` to it? This would change the interface since it'll remove `url` and add `method` that's from `CustomRequestInit`.

Reply:
Oh shoot, didn't mean to exclude url. Missed that it is not included by the new supertype. Will fix.